libparsec/crates/platform_device_loader/Cargo.toml

```diff
@@ -7,6 +7,7 @@ homepage.workspace = true
 license.workspace = true
 version.workspace = true
 repository.workspace = true
+autotests = false
```

Comment:
I'd rather stay consistent since we don't do this with the other crates

Can you instead create a `tests/unit/` folder containing all the tests files (and the `mod.rs` you added)